assets/src/edit-story/elements/text/display.js

```diff
@@ -227,6 +226,17 @@ function TextDisplay({
         }
       : border;
 
+  borderRadius =
+    previewMode && borderRadius
+      ? {
+          ...borderRadius,
+          topLeft: dataToEditorX(borderRadius.topLeft),
+          topRight: dataToEditorX(borderRadius.topRight),
+          bottomLeft: dataToEditorX(borderRadius.bottomLeft),
+          bottomRight: dataToEditorX(borderRadius.bottomRight),
+        }
```

Comment:
This is now duplicated three times. Also, what's left in the `borderRadius` object to spread in line 232?